src/server/routes/admin/nonces.ts

```diff
@@ -0,0 +1,169 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { eth_getTransactionCount, getRpcClient } from "thirdweb";
+import {
+  lastUsedNonceKey,
+  recycledNoncesKey,
+  sentNoncesKey,
+  splitLastUsedNonceKey,
+} from "../../../db/wallets/walletNonce";
+import { getChain } from "../../../utils/chain";
+import { normalizeAddress } from "../../../utils/primitiveTypes";
+import { redis } from "../../../utils/redis/redis";
+import { thirdwebClient } from "../../../utils/sdk";
+import { standardResponseSchema } from "../../schemas/sharedApiSchemas";
+import { walletWithAddressParamSchema } from "../../schemas/wallet";
+
+export const responseBodySchema = Type.Object({
+  result: Type.Array(
+    Type.Object({
+      walletAddress: Type.String({
+        description: "Backend Wallet Address",
+        examples: ["0xcedf3b4d8f7f1f7e0f7f0f7f0f7f0f7f0f7f0f7f"],
+      }),
+      onchainNonce: Type.String({
```

Comment:
We should be consistent representing nonces everywhere. Besides the fact that Redis requires strings, IMO I like to use numbers everywhere. WDYT? e.g. this would be a `Type.Integer()`

Or better yet a common type we'd reuse:
```typescript
export const WalletNonceSchema = Type.Integer({
  description: "A wallet nonce",
  examples: [123],
})

// Usage
onchainNonce: {
  ...WalletNonceSchema,
  description: "Last mined nonce"
}
```